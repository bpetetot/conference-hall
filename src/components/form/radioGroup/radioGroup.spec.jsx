/* eslint-env jest */
import React from 'react'
import RadioGroup from './index'

const props = {
  value: 'value',
  name: 'name',
}

describe('components/radiogroup', () => {
  it('should render', () => {
    expect(<RadioGroup {...props}><span>child1</span><span>child2</span></RadioGroup>)
      .toMatchSnapshot()
  })
})
