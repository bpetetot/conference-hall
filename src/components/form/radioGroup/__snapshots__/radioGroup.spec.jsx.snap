// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/radiogroup should render 1`] = `
<RadioGroup
  inline={false}
  name="name"
  noError={false}
  value="value"
>
  <span>
    child1
  </span>
  <span>
    child2
  </span>
</RadioGroup>
`;
